kolibri/plugins/device/assets/src/views/AvailableChannelsPage/index.vue

```diff
@@ -210,6 +233,19 @@
       },
     },
     watch: {
+      installedChannelsWithResources: {
```

Comment:
I'm thinking this might not be necessary to fix #6366. I'm worried that channels might move around unexpectedly if other Tasks get started by other admins, or by the same user.

Let's just make sure that more "permanent" ordering on the "Channel View" is maintained in the "Import View".

Reply:
That makes sense. I've removed the unnecessary sections!